tinyagentos/todo/migration.py

```
@@ -0,0 +1,130 @@
+        title = doc.get("title", "")
+        entries = doc.get("entries", [])
+
+        existing_lists = await todo_store.list_lists(
+            owner, include_archived=True
+        )
+
+        # -- primary idempotency check: exact source-doc match -----------
+        # If a todo list already has migrated_from == old_id we know this
+        # exact source was migrated on a previous run.
+        migrated_match = [
+            l for l in existing_lists
```

Ruff (0.15.21)

[error] Ambiguous variable name: `l`

(E741)